custom/icds_reports/templates/icds_reports/icds_app/mobile/busy.html

```diff
@@ -0,0 +1,5 @@
+{% load staticfiles %}
+<div class="position-fixed w-100 h-100vh d-flex justify-content-center align-items-center"
+     style="background: rgba(0,0,0,0.5); top: 0; z-index: 99;">
```

Comment:
Is this going to interact properly with `.sideNavigation`?   https://github.com/dimagi/commcare-hq/blob/be1b3cc95752e2c7d347d526e91bf73bf2139054/custom/icds_reports/static/icds_reports/mobile/css/navigation-menu.less#L13

z index is an especially scary property to do as an inline style. Best practice is to use variables for z-index and to define them all in the same file - [here's where HQ does it](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/includes/variables.less#L103-L112). It should be straightforward to add a `variables.less` file for the dashboard and move this into there.

Reply:
Nice catch and thanks for the feedback. Turns out this wasn't even needed so I just got rid of it.

Reply:
> Turns out this wasn't even needed

Sweet, the fewer z indices the better!